spec/rubocop/cop/style/redundant_freeze_spec.rb

```diff
@@ -23,6 +23,11 @@
   it_behaves_like 'immutable objects', '1.5'
   it_behaves_like 'immutable objects', ':sym'
   it_behaves_like 'immutable objects', ':""'
+  it_behaves_like 'immutable objects', "ENV['foo']"
+  it_behaves_like 'immutable objects', "'foo'.count"
+  it_behaves_like 'immutable objects', '1 + 2'
+  it_behaves_like 'immutable objects', '2 > 1'
```

Comment:
Is this case meaning `(2 > 1).freeze`? If so, it seems necessary to use parentheses like `(2 > 1)`.

Reply:
This is no longer part of the PR. New PR to come soon.